handbook/engineering/code-intelligence/index.md

```diff
@@ -166,10 +175,8 @@ Our team is growing and to help our new teammates have the best onboarding exper
 - [Alerts owned by this team](https://sourcegraph.com/search?q=repo:%5Egithub.com/sourcegraph/sourcegraph%24+file:monitoring/.*+%7B:%5B_%5D%2C+Owner:+ObservableOwnerCodeIntel%2C+:%5B_%5D%7D+OR+%28:%5B_%5D%2C+ObservableOwnerCodeIntel%29+count:1000&patternType=structural)
 - [OpsGenie rotation](https://sourcegraph.app.opsgenie.com/teams/dashboard/d0c10593-3edd-4d7e-8d1b-2ad29afeaa71)
 
-## Hiring status
-
-_Updated 2020-08-01_
+## Growth plan
 
-We are hiring for these roles:
+_Updated 2020-09-21_
 
-- +1 [Software Engineer - Code intelligence](https://about.sourcegraph.com/handbook/engineering/hiring/software-engineer-code-intelligence)
+We are hiring an [Engineering Manager](../hiring/engineering-manager-code-intelligence.md) to lead and grow this team.
```

Comment:
I'm assuming this means we're freezing the role for an engineer until we onboard a new manager?

Reply:
We will process all existing applications, but will temporarily delist those application forms (@aidaeology has a TODO) until we have a manager who can own it. Obviously if we come across an amazing candidate who we want to interview and/or hire, we will do so even if we don't already have a manager, but that is not the expected outcome.

Reply:
Does this also apply to the full-stack code intel position?

Reply:
@tsenart will continue processing candidates who apply to the full stack position since the Cloud team is also looking to hire for that role. It is possible we will find someone great for the code intel team via that hiring pipeline.